backend/services/messaging/message_service.py

```diff
@@ -450,7 +451,44 @@ def resend_email_validation(user_id: int):
         SMTPService.send_verification_email(user.email_address, user.username)
 
     @staticmethod
-    def _parse_message_for_username(message: str) -> List[str]:
+    def _get_managers(message: str, project_id: int) -> List[str]:
+        parser = re.compile(r"((?<=#)\w+|\[.+?\])")
+        parsed = parser.findall(message)
+
+        prj = None
+        if "author" in parsed or "managers" in parsed:
+            prj = Project.query.get(project_id)
+
+        if prj is None:
+            return []
+
+        prj_usernames = [prj.author.username]
+
+        if "managers" not in parsed:
+            return prj_usernames
+
+        teams = [t for t in prj.teams if t.role == TeamRoles.PROJECT_MANAGER.value]
+        team_members = [
+            [
+                u.member.username
+                for u in t.team.members
+                if u.function == TeamMemberFunctions.MANAGER.value
```

Comment:
On the case of Project Manager teams, we should get all active users, not only the team managers.